src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -4,6 +4,7 @@
 # --------------------------------------------------------------------------------------------
 
 import re
+import time
```

Comment:
where's is this time used.

Reply:
time is used to wait for a while before sending retry request